<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sanguosha</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStart_Game"/>
    <addaction name="actionReplay"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionStart_Server"/>
    <addaction name="actionGeneral_Overview"/>
    <addaction name="actionCard_Overview"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Hide_Menu"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionEnable_Hotkey"/>
    <addaction name="actionView_Discarded"/>
    <addaction name="actionView_distance"/>
    <addaction name="actionServerInformation"/>
    <addaction name="actionSurrender"/>
    <addaction name="actionKick"/>
    <addaction name="actionSaveRecord"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_audiere"/>
    <addaction name="actionAbout_libircclient"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../resource/images/sanguosha.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionStart_Server">
   <property name="text">
    <string>Start Server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStart_Game">
   <property name="text">
    <string>Start Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGeneral_Overview">
   <property name="text">
    <string>General Overview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCard_Overview">
   <property name="text">
    <string>Card Overview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEnable_Hotkey">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Hotkey</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionView_Discarded">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Discarded</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionNever_Nullify_My_Trick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Never Nullify My Trick</string>
   </property>
  </action>
  <action name="actionView_distance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View distance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout_audiere">
   <property name="text">
    <string>About audiere</string>
   </property>
  </action>
  <action name="actionServerInformation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Server information</string>
   </property>
  </action>
  <action name="actionKick">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Kick</string>
   </property>
  </action>
  <action name="actionReplay">
   <property name="text">
    <string>Replay</string>
   </property>
  </action>
  <action name="actionSaveRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save battle record</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionShow_Hide_Menu">
   <property name="text">
    <string>Show/Hide Menu</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionSurrender">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Surrender</string>
   </property>
  </action>
  <action name="actionAbout_libircclient">
   <property name="text">
    <string>About libircclient</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/images/sanguosha.qrc"/>
 </resources>
 <connections/>
</ui>
